data/tlg0057/tlg012/__cts__.xml

```diff
@@ -2,6 +2,6 @@
   <ti:title xml:lang="lat">De ossibus ad tirones</ti:title>
   <ti:edition urn="urn:cts:greekLit:tlg0057.tlg012.1st1K-grc1" workUrn="urn:cts:greekLit:tlg0057.tlg012" xml:lang="grc">
     <ti:label xml:lang="lat">De ossibus ad tirones</ti:label>
-    <ti:description xml:lang="mul">Galen, De ossibus ad tirones, Claudii Galeni Opera Omnia, Kühn, Volume 2, 1821</ti:description>
+    <ti:description xml:lang="mul">Galen. Claudii Galeni Opera Omnia, Volume 2. Kühn, Karl Gottlob, editor. Leipzig: Cnobloch, 1821.</ti:description>
```

Comment:
I think if you are removing the Latin (Leipzig not Lipsiae) then this would be Knobloch not Cnobloch, which is the Latin rendering of the proper name.
See the LCL: Kühn, C. G. Claudii Galeni Opera Omnia. Leipzig: Knobloch, 1821–1833.
https://www.loebclassics.com/view/LCL523/2016/pb_LCL523.lxix.xml?readMode=recto

Reply:
hey @lcerrato look at that,I will go change that and then resubmit this pull request.

Reply:
actually @lcerrato I think I will just do one last pass at the end since I have a number of files committed and a number not, but I will change it moving forward.